lib/rails_admin/config/fields/types/polymorphic_association.rb

```diff
@@ -16,7 +16,7 @@ class PolymorphicAssociation < RailsAdmin::Config::Fields::Types::BelongsToAssoc
           # association checks that any of the child models are included in
           # configuration.
           register_instance_option :visible? do
-            associated_model_config.length > 0
+            !associated_model_config.empty?
```

Comment:
I'll be honest with you... I didn't like this but also didn't take the time to look into what associated_model_config returned and so played it safe :)
Looking at it now, it seems like `association.klass` should always return an array so it should be safe to use any?

I'll make this change now.